README.md

```diff
@@ -59,42 +59,6 @@ assert_eq!(
 );
 ```
 
-### Future Version (1.3)
```

Comment:
It would be nice if someone looking at the tutorials at the tag icu-1.3.0 would see 1.3 tutorials

Reply:
We should do the same as with 1.2, have a 1.3 tutorials PR ready to merge and then we can tag that.